sdk/src/test/java/com/uid2/sdk/data/TestData.kt

```diff
@@ -12,4 +15,54 @@ object TestData {
     const val VALID_IDENTITY_REFRESH_FROM = 321L
     const val VALID_IDENTITY_REFRESH_EXPIRES = 654321L
     const val VALID_IDENTITY_REFRESH_RESPONSE_KEY = "response key"
+
+    // A valid json representation, but missing the required "status" parameter.
+    const val INVALID_REFRESH = "{\"body\":{\"advertising_token\":\"token\",\"refresh_token\":\"refresh\",\"identity_expires\":123456,\"refresh_from\":321,\"refresh_expires\":654321,\"refresh_response_key\":\"response key\"}}"
+
+    // A valid, yet made up, json representation. The body contains the VALID_IDENTITY example shared above.
+    const val VALID_REFRESH = "{\"body\":{\"advertising_token\":\"token\",\"refresh_token\":\"refresh\",\"identity_expires\":123456,\"refresh_from\":321,\"refresh_expires\":654321,\"refresh_response_key\":\"response key\"},\"status\":\"success\"}"
+    const val VALID_REFRESH_STATUS = "success"
+
+    const val VALID_REFRESH_OPT_OUT = "{\"status\":\"optout\",\"message\":\"User opted out\"}"
+    const val VALID_REFRESH_EXPIRED_TOKEN = "{\"status\":\"expired_token\",\"message\":\"Token expired\"}"
+
+    // A Base64 encrypted refresh response that reports a status of "optout"
+    val REFRESH_TOKEN_OPT_OUT_ENCRYPTED = ResourceHelper.loadString("test-data/refresh-token-200-optout-encrypted.txt")
```

Comment:
I originally defined test data as strings within this class, but after adding support for decryption, this became a bit crazy. That's when I introduced the `ResourceHelper` to load from asserts (thanks Richard!). If people prefer this way, then I can migrate other hardcoded JSON from here back into actual assert files.

Reply:
My two cents are that I prefer to have test data in local files.  Keeps things isolated / stable and the tests more readable.